modules/ROOT/pages/custom-log-appender.adoc

```diff
@@ -89,31 +100,30 @@ remote debugging purposes by MuleSoft staff.
 </Configuration>
 ----
 
+
 == Enable Custom Log4j Configurations in CloudHub
 
-Once you've configured your Log4j appenders, deploy the application to CloudHub
-and tell CloudHub you are supplying your own Log4j configuration:
+After you configure the Log4j appender, deploy the application to CloudHub and follow these steps to disable CloudHub logs:
 
-. Log into https://anypoint.mulesoft.com/#/signin[Anypoint Platform] and click *CloudHub*.
-. Click an application to expand the details view:
+. From Anypoint Platform, select *Runtime Manager*.
+. Click *Applications* in the left menu.
+. Click the *Type* column to display the details pane for the application.
+. Click *Manage Application*.
+. In the *Settings* page, click *Disable CloudHub logs*:
 +
-image::chapps.png[CHApps]
+.The arrow shows the *Disable CloudHub logs* option in the *Settings* page.
+image::cloudhub-disable-logs.png[Disable CloudHub logs option in the Applications Settings page]
+. In the *Disabling CloudHub logs* confirmation windows, select the checkboxes to verify that you want to disable CloudHub Loggers the server group:
```

Comment:
Please reread and verify the wording of "...to verify that you want to disable CloudHub Loggers the server group:"